Compiled from "NestedExceptionUtils.java"
public abstract class org.springframework.core.NestedExceptionUtils {
  public org.springframework.core.NestedExceptionUtils();
    Code:
       0: aload_0
       1: invokespecial #1                  // Method java/lang/Object."<init>":()V
       4: return

  public static java.lang.String buildMessage(java.lang.String, java.lang.Throwable);
    Code:
       0: aload_1
       1: ifnull        43
       4: new           #2                  // class java/lang/StringBuilder
       7: dup
       8: invokespecial #3                  // Method java/lang/StringBuilder."<init>":()V
      11: astore_2
      12: aload_0
      13: ifnull        27
      16: aload_2
      17: aload_0
      18: invokevirtual #4                  // Method java/lang/StringBuilder.append:(Ljava/lang/String;)Ljava/lang/StringBuilder;
      21: ldc           #5                  // String ;
      23: invokevirtual #4                  // Method java/lang/StringBuilder.append:(Ljava/lang/String;)Ljava/lang/StringBuilder;
      26: pop
      27: aload_2
      28: ldc           #6                  // String nested exception is
      30: invokevirtual #4                  // Method java/lang/StringBuilder.append:(Ljava/lang/String;)Ljava/lang/StringBuilder;
      33: aload_1
      34: invokevirtual #7                  // Method java/lang/StringBuilder.append:(Ljava/lang/Object;)Ljava/lang/StringBuilder;
      37: pop
      38: aload_2
      39: invokevirtual #8                  // Method java/lang/StringBuilder.toString:()Ljava/lang/String;
      42: areturn
      43: aload_0
      44: areturn
}
